client/packages/system/src/Program/api/api.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  SortBy,
+  FilterBy,
+  DocumentRegistrySortFieldInput,
+} from '@openmsupply-client/common';
+import { ProgramDocumentFragment, Sdk } from './operations.generated';
+
+export type ListParams = {
+  first?: number;
+  offset?: number;
+  sortBy?: SortBy<ProgramDocumentFragment>;
+  filterBy?: FilterBy | null;
+};
+
+export const getProgramQueries = (sdk: Sdk) => ({
+  get: {
+    list: async ({
+      sortBy,
+    }: ListParams): Promise<{
+      nodes: ProgramDocumentFragment[];
+      totalCount: number;
+    }> => {
+      const result = await sdk.programs({
+        key:
+          (sortBy?.key as DocumentRegistrySortFieldInput) ??
+          DocumentRegistrySortFieldInput.DocumentType,
+        desc: sortBy?.isDesc,
+      });
+
+      return result?.documentRegistries;
```

Comment:
There are already some programs queries for for the list view. Did you copied it intentionally?

Reply:
the other ones are filtered for a specific patient.. which I can't use when enrolling a patient into a new program

Reply:
sorry got confused by the name, totally makes sense. Rename ProgramDocumentFragment to ProgramDocumentRegistryFragment though?

Reply:
I can update next time.. got a little merge trigger happy. though I'm concerned that we'll have xxDocumentRegistry things all over the place 🤷